docs/charts.md

```diff
@@ -203,6 +203,80 @@ Managing charts with `requirements.yaml` is a good way to easily keep
 charts updated, and also share requirements information throughout a
 team.
 
+#### Tags and Condition fields in requirements.yaml
+
+In addition to the other fields above, each requirements entry may contain 
+the optional fields `tags` and `condition`.
+
+All charts are loaded by default. If `tags` or `condition` fields are present, 
+they will be evaluated and used to control loading for the chart(s) they are applied to. 
+
+Condition - The condition field holds one or more YAML paths (delimited by commas). 
+If this path exists in the top parent's values and resolves to a boolean value, 
+the chart will be enabled or disabled based on that boolean value.  Only the first 
+valid path found in the list is evaluated and if no paths exist then the condition has no effect.
+
+Tags - The tags field is a YAML list of labels to associate with this chart. 
+In the top parent's values, all charts with tags can be enabled or disabled by 
+specifying the tag and a boolean value.
+
+````
+# parentchart/requirements.yaml
+dependencies:
+      - name: subchart1
+        repository: http://localhost:10191
+        version: 0.1.0
+        condition: subchart1.enabled, global.subchart1.enabled
+        tags:
+          - front-end
+          - subchart1
+      
+      - name: subchart2
+        repository: http://localhost:10191
+        version: 0.1.0
+        condition: subchart2.enabled,global.subchart2.enabled
+        tags:
+          - back-end
+          - subchart1
+
+````
+````
+# parentchart/values.yaml
+
+subchart1:
+  enabled: true
+tags:
+  - front-end: false
```

Comment:
Apparently they both work? I had created and tested with the 'subpop' chart and it uses the list with the dash. I have not tried it without and will go lookup what the difference means in YAML now ;)